### PR TITLE
UIDEXP-21: move getFileExtension from ui-data-import

### PR DESCRIPTION
**Pull request by Yuliia-Dovhal**
## Purpose:
Added getFileExtension function from ui-data-import to utils here because of reuse in scope of the [UIDEXP-21](https://issues.folio.org/browse/UIDEXP-21)

### CODE REVIEW

File: lib/utils/getFileExtension.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/** @param {File | { name: string }} file */
+export const getFileExtension = file => {
+  if (file && file.name) {
+    const fileExtensionRegExp = /\.(\w+)$/;
+    const [extension = null] = (file.name.match(fileExtensionRegExp) || []);
+
+    return extension && extension.toLowerCase();
```
Comment (VictorSoroka1) [inline lib/utils/getFileExtension.js]:
```suggestion
    return extension?.toLowerCase();
```

File: lib/utils/getFileExtension.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/** @param {File | { name: string }} file */
+export const getFileExtension = file => {
+  if (file && file.name) {
```
Comment (VictorSoroka1) [inline lib/utils/getFileExtension.js]:
```suggestion
  if (file?.name) {
```

File: lib/utils/tests/getFileExtension-test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  it,
+} from '@bigtest/mocha';
+
+import { getFileExtension } from '..';
+
+describe('getFileExtension', () => {
+  it('should return file extension', () => {
+    const testFile = new File([], 'test.csv');
+
+    expect(getFileExtension((testFile))).to.be.equal('.csv');
```
Comment (YevheniiMaltsev) [inline lib/utils/tests/getFileExtension-test.js]:
```suggestion
    expect(getFileExtension((testFile))).to.equal('.csv');
```
Please, check other places

File: lib/utils/getFileExtension.js (hunk 2 above)
Comment (Yuliia-Dovhal) [reply lib/utils/getFileExtension.js]: Looks much better, thanks!

File: index.js [hunk 4]
```diff
@@ -13,6 +13,7 @@ export {
   createUrl,
```
[the remark this answers is not in the document]
Comment (Yuliia-Dovhal) [reply index.js]: Good point, thanks!

**Pull request merged**